apps/rush-lib/src/utilities/VersionControl.ts

```diff
@@ -7,9 +7,9 @@ import * as child_process from 'child_process';
  * @public
  */
 export default class VersionControl {
-  public static getChangedFolders(targetBranch?: string): string[] {
+  public static getChangedFolders(targetBranch?: string): Array<string | undefined> | undefined {
```

Comment:
>undefined [](start = 73, length = 9)

Why are we doing this?  Seems like a bad design

Reply:
I thought so too, but I wasn't trying to change any behavior in this PR.

---
In reply to: [142773890](https://github.com/Microsoft/web-build-tools/pull/385#discussion_r142773890) [](ancestors = 142773890)